api/controllers/game/load-fixture-gamestate.js

```diff
@@ -0,0 +1,85 @@
+const MoveType = require('../../../utils/MoveType.json');
+const GamePhase = require('../../../utils/GamePhase.json');
+const DeckIds = require('../../../utils/DeckIds.json');
+
+module.exports = async function (req, res) {
+  try {
+    const game = await Game.findOne({ id: req.session.game })
+      .populate('gameStates')
+      .populate('p0')
+      .populate('p1');
+    const { addCardId, publishGameState, saveGamestate, convertStrToCard } = sails.helpers.gamestate;
+    const p0Hand = req.body.p0Hand?.map((card) => addCardId(card));
```

Comment:
Feels like you should destructure body higher up to reference them all more directly.

Reply:
The main reason I didn't destructure was because I wanted to keep the same names after I had added the ID to each card. Is there a way to do that with destructuring?